.eslintrc

```diff
@@ -41,7 +41,8 @@
     "@shopify/jsx-prefer-fragment-wrappers": "off",
     "@shopify/no-ancestor-directory-import": "error",
     "@shopify/react-prefer-private-members": "off",
-    "@typescript-eslint/array-type": ["error", {"defaultOption": "array"}],
+    "@typescript-eslint/array-type": ["error", {"default": "array"}],
+    "@typescript-eslint/naming-convention": "off",
```

Comment:
**question (non-blocking):** Should we configure this rule? Seems helpful to enforce naming conventions.